src/storage/src/mem_table.rs

```diff
@@ -575,6 +575,25 @@ impl<S: StateStoreWrite + StateStoreRead> LocalStateStore for MemtableLocalState
             prev_epoch
         );
     }
+
+    async fn try_flush(&mut self) -> StorageResult<()> {
```

Comment:
For state store other than hummock, the try_flush shall be no-op and no need to maintain spill offset. We only need to support write any time for hummock.

Reply:
so it should be `unreachable!()`

Reply:
Making it spillable is just making it able to call `try_flush`. Doing nothing in `try_flush` is acceptable. Other state store does not support writing a key for multiple times in an epoch and if spill happens we may get error. The easiest way is to do nothing in `try_flush` to not doing any spill.

Reply:
May change to no-op for the `try_flush` here?